### PR TITLE
Added a function to clean counterDB on counter

### PR DESCRIPTION
**Pull request by gabrielfedel**
I added a function to delete all counters from counterDB

### CODE REVIEW

**Comment by hhslepicka** [comment]
Do you have a use-case for that?

**Comment by gabrielfedel** [comment]
Yes, for the gui. I having problems with two consecutive runs. In the second run counterDB stay with old counters and I can't create news (with the same mnemonic, but with new parameters).

File: py4syn/utils/counter.py [hunk 1]
```diff
@@ -219,6 +219,9 @@ def enableCounter(mne):
     c = py4syn.counterDB[mne]
     c['enable'] = True
 
+def delCounters():
```
Comment (hhslepicka) [inline py4syn/utils/counter.py]: Please rename it to something like:
```
def clearCountersDB()
```

Del can lead to a different understanding of it.

**Comment by hhslepicka** [comment]
@gabrielfedel no problem. I just added a single review in there.
Please submit the requested change and I will merge it.

**Review by hhslepicka — CHANGES_REQUESTED** [review]

**Comment by gabrielfedel** [comment]
Thank you! Done!

**Review by hhslepicka — APPROVED** [review]

**Comment by hhslepicka** [comment]
LGTM!

**Pull request merged**